README.md

```diff
@@ -214,64 +224,323 @@ Thus, this allows you to have many vendored tarballs by using the `--tag` parame
-      --update <UPDATE>
-          Update dependencies or not [default: true] [possible values: true, false]
-      --filter <FILTER>
-          EXPERIMENTAL: Reduce vendor-tarball size by filtering out non-Linux dependencies. [default: false] [possible values: true, false]
-      --outdir <OUTDIR>
-          Where to output vendor.tar* and cargo_config
-      --color <WHEN>
-          Whether WHEN to color output or not [default: auto] [possible values: auto, always, never]
-      --i-accept-the-risk <I_ACCEPT_THE_RISK>
-          A list of rustsec-id's to ignore. By setting this value, you acknowledge that this issue does not affect your package and you should be exempt from resolving it.
-      --respect-lockfile <RESPECT_LOCKFILE>
-          Respect lockfile or not if it exists. Otherwise, regenerate the lockfile and try to respect the lockfile. [default: true] [possible values: true, false]
-      --versioned-dirs <VERSIONED_DIRS>
-          Whether to use the `--versioned-dirs` flag of cargo-vendor. [default: true] [possible values: true, false]
-  -h, --help
-          Print help (see more with '--help')
-  -V, --version
-          Print version
+
+> [!IMPORTANT]
+> Overall, both methods are not perfect. But with the right combination of `--custom-root` and `--no-root-manifest`,
+> it gets easier if you're able to find that combination.
+> You can see more in the [./cargo/tests/behaviour.rs](./cargo/tests//behaviour.rs) for
+> such combinations.
+
+Filtering is not supported in this method. See more in the [./cargo_vendor.service](./cargo_vendor.service) file or the
+[Parameters](#parameters) section below.
+
+# Parameters
+
+The following are the parameters you can use with this utility:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<service name="cargo_vendor">
+   <summary>OBS Source Service to vendor all crates.io and dependencies for a Rust project</summary>
+   <description><![CDATA[This service extracts a Rust application source,
+  searches for a Rust application containing a Cargo.toml file,
+  download all crates.io and dependecies,
+  and creates a vendor.tar[.<tar compression>] to be committed allowing fully offline
+  builds of Rust applications.]]></description>
+   <parameter name="strategy">
+      <description>Legacy argument, no longer used. Values: cargo_vendor. Default: cargo_vendor</description>
+   </parameter>
+   <parameter name="method">
+      <description>Whether to use vendor or the registry. Default: vendor</description>
+      <allowedvalue>registry</allowedvalue>
+      <allowedvalue>vendor</allowedvalue>
+   </parameter>
+   <parameter name="src">
+      <description>Where to find sources. Source is either a directory or a source tarball AND cannot be both. Aliases: srctar, srcdir</description>
+   </parameter>
+   <parameter name="outdir">
+      <description>Where to output vendor.tar* and cargo_config if method is vendor and registry.tar* if method is registry.</description>
```

Comment:
Should say that `osc service` appends this automatically for you.

Reply:
I have a feeling we can make this utility standalone, although, it won't hurt to append that `osc service` appends it for us.